src/run.ts

```diff
@@ -50,20 +53,25 @@ export async function run() {
          runnerTempDirectory,
          `kubeconfig_${Date.now()}`
       )
+
       core.debug(`Writing kubeconfig to ${kubeconfigPath}`)
+
       const cmd = [
-         'aks',
-         'get-credentials',
-         '--resource-group',
-         resourceGroupName,
-         '--name',
-         clusterName,
-         '-f',
-         kubeconfigPath
-      ]
-      if (subscription) cmd.push('--subscription', subscription)
-      if (admin) cmd.push('--admin')
-      if (publicFqdn) cmd.push('--public-fqdn')
+            resourceType=='fleet' ? 'fleet':'aks',
```

Comment:
```suggestion
            resourceType.tolower() == 'microsoft.containerservice/fleets' ? 'fleet' : 'aks',
```

^ something like this?

Reply:
nit: please leave the indentation as it was